src/server/es/aggs.js

```diff
@@ -581,7 +581,7 @@ export const textAggregation = async (
   // don't add missing alias to numeric field by default
   // since the value of missing alias is a string
   if (config.esConfig.aggregationIncludeMissingData && !isNumericField) {
-    missingAlias = { missing: config.esConfig.missingDataAlias };
```

Comment:
why is this changing? this `missingDataAlias` should be configurable via config

Reply:
if guppy no longer returns `no data` (or the missing data alias as defined in confog) for missing data, some envs may be affected, for example https://github.com/uc-cdis/gitops-qa/blob/master/qa-covid19.planx-pla.net/portal/gitops.json#L149 this one depends on that missing data alias value

We need to make sure this is tested

Reply:
I think I have commented here but got no answer, can you elabrate? @thanh-nguyen-dang

Reply:
ES does not support missing configuration anymore. We need to use different way {missing_bucket: true,  order: 'desc'} to configure the field with missing data. 
This part will handle the missing data
https://github.com/uc-cdis/guppy/pull/200/files#diff-5db64ea5481e180cbf163669ddb74578c387c32bafcbb7a4dccc8d57c1dc1102R685-R689